firmware/common/message.hpp

```diff
@@ -1013,6 +1026,29 @@ class FSKConfigureMessage : public Message {
     const uint32_t progress_notice;
 };
 
+class FSKRxConfigureMessage : public Message {
+   public:
+    constexpr FSKRxConfigureMessage(
+        const fir_taps_real<24> decim_0_filter,
```

Comment:
Question for myself: does the UI layer normally work with these types?

Reply:
It definitely doesn't. I'm not sending them at the UI level. I'm going to eventually rework this because I'm planning on finishing this app next so I think the parameters are probably going to change anyway.

Reply:
This was a hard copy from NBFM message configuration, but I'm finding this isn't the way I want to go, but it was left in here because I hadn't completed the fsk app when I started working on ble.